examples/evm-to-evm-erc20-contract-call/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@buildwithsygma/evm-to-evm-erc20-contract-call",
+  "version": "0.1.0",
```

Comment:
out of curiosity, why 0.1 as version for the example?

Reply:
previous examples had `0.1.0` and we didn't update package versions for examples, and it doesn't matter cause they are not meant to be published (atleast for now)

Reply:
yeah I just realized that